Makefile

```diff
@@ -149,6 +150,11 @@ cfn-lint/install:
 yq/install:
 	@ $(MAKE) install/pip/$(@D) PYPI_PKG_NAME=$(@D)
 
+bump2version/install:
+	@ $(MAKE) install/pip/$(@D) PYPI_PKG_NAME=$(@D) CUSTOM_VERSION_CMD="bumpversion -h | grep 'bumpversion: v'"
```

Comment:
CUSTOM_VERSION_CMD is being used ...

Reply:
Right, but that's not what I am referring to. Make will pass through *any* var that you set like this. So yes, this invocation is passing `CUSTOM_VERSION_CMD`. But the _default_ on `install/pip` is not, because that's not the name of the target where it is used. The target name is `install/pip/%`.